django/contrib/admin/static/admin/css/base.css

```diff
@@ -49,6 +49,10 @@ html[data-theme="light"],
     --selected-bg: #e4e4e4; /* E.g. selected table cells */
     --selected-row: #ffc;
 
+    --selector-button-bg: #e9e9ed;
+    --selector-button-fg: #525252;
+    --selector-button-hover-bg: #d0d0d7;
```

Comment:
Instead of defining new styling, I think it might be worth reusing the existing button styling:

<img width="1518" height="941" alt="Screenshot from 2025-08-05 10-26-51" src="https://github.com/user-attachments/assets/b8b8e2d1-6316-4474-9a0a-c1a253c28413" />

Notice that there is a "Reset password" button, I think we can have consistent styling with this (using `--button-bg` and not bold text).
Folks then have less button variations to learn, and if we make accessibility improvements to buttons, we have less variations to worry about

The @django/accessibility team might have an opinion here